bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -563,7 +563,21 @@ public void sendCommand(String io, String command, String params, String url) {
 
         Boolean result = sendCommandInternal(io, command, params, url);
         if (!result) {
-            sendCommandInternal(io, command, params, url);
+            scheduler.schedule(() -> {
+                repeatSendCommandInternal(io, command, params, url, thingConfig.getRetries());
+            }, thingConfig.getRetryDelay(), TimeUnit.MILLISECONDS);
+        }
+    }
+
+    private void repeatSendCommandInternal(String io, String command, String params, String url, int retries) {
+        logger.info("Retrying command, retries left: {}", retries);
+        Boolean result = sendCommandInternal(io, command, params, url);
+        if (result != null && !result) {
+            if (retries > 0) {
+                scheduler.schedule(() -> {
+                    repeatSendCommandInternal(io, command, params, url, retries - 1);
```

Comment:
cosmetic:  repeatSendCommandInternal(io, command, params, url, --retries);

Reply:
--retries won't compile. I guess this is due to the usage in the lambda which cannot change a variable outside of its scope.

Reply:
OK